### PR TITLE
Update communication-tools.md

### PR DESCRIPTION
**Pull request by kniz-raft**
### This pull request changes...

*Added links to research privacy PPT and link to Mural links*

## FUNCTIONALITY
*Describe the functionality*
*Addresses Issue #*

## TO TEST

*List the steps to test the PR*

### This pull request is ready to merge when...

- [ ] Meets acceptance criteria for the issue 
- [ ] Code is reviewed by someone other than the original author
- [ ] Code is tested 
- [ ] Experience is approved by UX  
- [ ] Meets [a11y checklist](https://github.com/HHS/TANF-app/blob/main/docs/a11y/how-18f-will-test-a11y.md)
- [ ] Meets [Raft's Manual QASP Checklist](https://teams.microsoft.com/l/file/B5285301-13CA-46FD-A723-AFFD578C224A?tenantId=d58addea-5053-4a80-8499-ba4d944910df&fileType=docx&objectUrl=https%3A%2F%2Fhhsgov.sharepoint.com%2Fsites%2FTANFDataPortalOFA%2FShared%20Documents%2FGeneral%2FManual%20Testing%20Practices_Issue173.docx&baseUrl=https%3A%2F%2Fhhsgov.sharepoint.com%2Fsites%2FTANFDataPortalOFA&serviceName=teams&threadId=19:f769bbcb029f4f02b55ae7fad90e310d@thread.skype&groupId=41f194a6-c1d3-4680-933e-c8ee7d17e287):lock: (only applicable for PR to main HHS)
- [ ] Documentation is updated 
  - OpenAPI  
  - Readme
  - Entity Relationship Diagram (ERD)

### CODE REVIEW

**Comment by kniz-raft** [comment]
@lfrohlich Can you take a look at this one too?

**Comment by lfrohlich** [comment]
Took a look and I want to make some updates to the Weekly Ship section. THanks for adding the links!

File: docs/How-We-Work/team-charter/communication-tools.md [hunk 1]
```diff
@@ -9,18 +9,18 @@
...
 
 
 ## Mural  
 
 - Currently, only Lauren has member access. Others can edit Murals as visitors after entering the password. Ask Lauren for the password.
-- [Mural links can be found here](NEED LINK)
+- [Mural links can be found here](https://teams.microsoft.com/l/channel/19%3Af769bbcb029f4f02b55ae7fad90e310d%40thread.skype/tab%3A%3Af5bd4b6f-f448-42a9-8437-073c4fbf556c?groupId=41f194a6-c1d3-4680-933e-c8ee7d17e287&tenantId=d58addea-5053-4a80-8499-ba4d944910df)🔒
  
 
 ## Weekly Ship Email
```
Comment (lfrohlich) [inline docs/How-We-Work/team-charter/communication-tools.md:21]:
```suggestion
## Bi-weekly Ship Email 
```

File: docs/How-We-Work/team-charter/communication-tools.md [hunk 2]
```diff
@@ -9,18 +9,18 @@
...
-- [Mural links can be found here](NEED LINK)
+- [Mural links can be found here](https://teams.microsoft.com/l/channel/19%3Af769bbcb029f4f02b55ae7fad90e310d%40thread.skype/tab%3A%3Af5bd4b6f-f448-42a9-8437-073c4fbf556c?groupId=41f194a6-c1d3-4680-933e-c8ee7d17e287&tenantId=d58addea-5053-4a80-8499-ba4d944910df)🔒
  
 
 ## Weekly Ship Email 
 
-- [Weekly ship email history lives in 18F’s Drive](https://docs.google.com/document/d/1QHSdnv4E68FBUC3vkHApgJIJ3zP2C5v9RUNszoeyHZk/edit) 
+- [Weekly ship email history lives in 18F’s Drive](https://docs.google.com/document/d/1QHSdnv4E68FBUC3vkHApgJIJ3zP2C5v9RUNszoeyHZk/edit)🔒 
 - Goal: To keep stakeholders’ informed of the team's work 
 - Format: Compiled and sent every Monday evening by Selena. Bulleted summary of our team’s. Includes work done by OFA, 18F, and Raft towards our shared goals.
```
Comment (lfrohlich) [inline docs/How-We-Work/team-charter/communication-tools.md:25]:
```suggestion
- Format: Compiled and sent every every other week by Lauren. Bulleted summary of our team’s work based on Sprint Review summaries. Includes work done by OFA, 18F, and Raft towards our shared goals.  
```

**Review by lfrohlich — COMMENTED** [review]
Updated ship

**Review by lfrohlich — APPROVED** [review]
Thanks for updating!

**Pull request merged**